osquery/tables/system/posix/smbios_utils.cpp

```diff
@@ -551,8 +551,7 @@ void genSMBIOSMemoryDeviceMappedAddresses(size_t index,
   results.push_back(std::move(r));
 }
 
-std::string dmiString(uint8_t* data, uint8_t* address, size_t offset) {
```

Comment:
Looks like address argument is not used anymore.

Reply:
Good catch.  I'll update.